docs/ref-cache.md

```diff
@@ -76,9 +76,28 @@ const options = createMutationOptions(mutationOptions);
 apolloClient.mutate(options);
 ```
 
-> NOTE: Cache helpers currently support only GraphQL Queries that return arrays.
-> For example `getUsers():[User]`.
-> When working with single objects returned by Queries we usually do not need use any helper as Query will be updated automatically on every update
+### Pagination and relationships
+
+It may be necessary to update a field within the query and not the entire query, for example with relationships. In order to do this, it is necessary to provide the name of the field being updated with the `returnField` parameter.
```

Comment:
```suggestion

Offix by default assumes that object returned by mutations will contain just your data. In situations when you need to wrap your data into some container to provide pagination and other information offix will require an additional parameter. For example when API returns TaskPage etc. we actual data might be returned over `items`.

When this query will be cached it may be necessary to update a field within the query and not the entire query, for example with relationships. In order to do this, it is necessary to provide the name of the field being updated with the `returnField` parameter.
```

Reply:
Thanks, I struggled with this a little